apps/generator/docs/generator-template.md

```diff
@@ -643,10 +643,555 @@ Temperature drop detected 66943992 sent to temperature/dropped
 Temperature rise detected 66943992 sent to temperature/risen
 ```
 
+## Creating a Java template
+
+Looking for more?
+
+This section will guide you in creating a flexible MQTT-supported template that will generate a **Java** client from the template and the AsyncAPI document above.
+The follow steps will be similar to the [initial steps](https://www.asyncapi.com/docs/tools/generator/generator-template#overview-of-steps) but with a few differences. This will help developers practice with AsyncAPI generator tool using statically typed language **Java**.
```

Comment:
```suggestion
The following steps are similar to the [initial steps](https://www.asyncapi.com/docs/tools/generator/generator-template#overview-of-steps) but with a few differences. This will help developers practice with AsyncAPI generator tool using statically typed language **Java**.
```